src/calculator.js

```diff
@@ -1,48 +1,29 @@
+module.exports = exports;
 exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
```

Comment:
&#39;x&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule